ui/pages/destinations/grantAccessContent.js

```diff
@@ -0,0 +1,151 @@
+import useSWR, { useSWRConfig } from 'swr'
```

Comment:
This file should be in `components` and not under `pages`. Even better would be to put it in the `destinations/index.js` page since it isn't that much code and this is the only place where it is used.

Reply:
I don't know if it is good idea to put it in the `destinations/index.js` page because I think it would make the `destinations/index,js` too long to maintain. But I think it is good idea to move it to `component`